app/components/views/ProposalDetailsPage/ProposalDetails.jsx

```diff
@@ -43,6 +43,11 @@ const ProposalDetails = ({
   goBackHistory,
   eligibleTicketCount
 }) => {
+  walletEligibleTickets = walletEligibleTickets.map((et, i) => {
```

Comment:
I would suggest moving this to GovernanceAction.js, to keep all proposal data logic in one place.

Reply:
I thought about that. The reason I didn't do it, is to avoid saving duplicated info into our `vote.json` file

Reply:
Oh, I see that's was the initial issue, the extra info